controls/radspreadsheet/howto/customize-selection.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Customize Selection
+page_title: Customize Selection
+description: Customize Selection
+slug: radspreadsheet-howto-customize-selection
+tags: customize,selection
+published: True
+position: 0
+---
+
+# Customize Selection
+
+RadSpreadsheet exposes several properties that allow you control the way the selection in the control is rendered. This article will describe the available properties and show you how to work with them.
+
+## 
+
+* **SelectionStroke**: Gets or sets the stroke of the selection. This is a *dependency property* and it is of type *Brush*.
```

Comment:
I noticed that you are using different styles when describing properties. 

"Gets or sets the stroke of the selection. This is a *dependency property* and it is of type *Brush*."
"It is a property of the RowColumnHeadingBase type. It is an enum type and could have the following values:"

I think it is kinda okay, but you can consider the option to choose a single style. We could discuss this if you like.